<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1167</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1181</width>
      <height>641</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame#frame_3 {
/*background-color: red;*/
	/*background-image: url(:/imgs/home/background_home.png);*/
	/*background-image:url(:/imgs/home/background_main3.png);*/
	background-image: url(:/imgs/home/background_home.png);
background-repeat: no-repeat;
  
  background-position: center;

}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1161</width>
       <height>601</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/*
QWidget {
background-image: url(&quot;:/imgs/register_page/image.png&quot;);
}

QWidget widget {
background-image:none;
}
*/
/*
QWidget {
background-color: red;
border-color: black;
border-width: 1px;
border-style: solid;
}
*/</string>
     </property>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>180</y>
        <width>311</width>
        <height>351</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/*background-color: #e8e8e6;*/
background-color: white;
border-radius: 15px;</string>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>10</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font-weight: bold;
color: black;
font-size: 40px;
border-radius: 1px;</string>
       </property>
       <property name="text">
        <string>User</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_3">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>281</y>
         <width>291</width>
         <height>62</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-width: 2px;
border-color: #112f91;
border-style: solid;
color: #112f91;
font-weight: bold;
border-radius: 10px;

</string>
          </property>
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #112f91;
color: white;
font-weight: bold;
border-radius: 5px;
</string>
          </property>
          <property name="text">
           <string>Register</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>291</width>
         <height>271</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 3px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resources.qrc">:/imgs/home/customer_home.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>-20</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;
color: black;
font-weight:bold;
font-size: 25px;</string>
       </property>
       <property name="text">
        <string>Buyer</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignHCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>180</y>
        <width>311</width>
        <height>351</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/*
border-color: red;
border-style: solid;
border-width: 1px;
background-image: none;
background-color: white;

background-color: #e8e8e6;*/
background-color: white;
border-radius: 10px;
border-radius: 15px;</string>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>90</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font-weight: bold;
color: black;
font-size: 40px;
border-radius:1px;

</string>
       </property>
       <property name="text">
        <string>Seller</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>291</y>
         <width>291</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="seller_login">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">/*background-color: #F10000;*/
border-width: 2px;
border-color: #F10000;
border-style: solid;
color: #F10000;
font-weight: bold;
border-radius: 10px;
</string>
          </property>
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="seller_register">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #F10000;
color: white;
font-weight: bold;
border-radius: 10px;
</string>
          </property>
          <property name="text">
           <string>Create New Seller Account</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>291</width>
         <height>271</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resources.qrc">:/imgs/home/seller_homee.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>-20</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;
color: black;
font-weight:bold;
font-size: 25px;</string>
       </property>
       <property name="text">
        <string>Seller</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignHCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>891</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 25px;
font-weight: bold;
background-color: #2E4F4F;
color: white;
border: none;

</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;SuperMarket Online Shopping&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>20</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/imgs/home/homeee.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>120</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-weight:bold;
font-size: 30px;</string>
      </property>
      <property name="text">
       <string>Join us As</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1167</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
